src/main.rs

```diff
@@ -90,10 +90,10 @@ fn main() {
             .last(true)
             .help("The command to run"))
         .arg(Arg::with_name("ttl")
-            .long("time_to_live")
-            .alias("ttl")
+            .long("time-to-live")
+            .visible_alias("ttl")
             .default_value("60s")
-            .help("Duration the cached result will be valid. Alias: --ttl"))
+            .help("Duration the cached result will be valid for"))
```

Comment:
I don't think "for" is helpful here, just leave the text as-is (without the "Alias").

Reply:
Done